src/modules/sync/components/sync-connected-sites.tsx

```diff
@@ -291,9 +295,13 @@ const SyncConnectedSitesSectionItem = ( {
 					{ pushState?.status && isPushing && (
 						<div className="flex items-center gap-2 max-w-full">
 							<Tooltip
-								text={ __(
-									'Push is in progress. We will send you an email when it is completed.'
-								) }
+								text={
+									pushBackupIsUploading( pushState?.status.key )
+										? __( 'Push is in progress. We will send you an email when it is completed.' )
+										: __(
+												"The push is in progress and will continue in the backend. We will send you an email once it's completed."
```

Comment:
```suggestion
												"The push is in progress and will continue running. We will send you an email once it's completed."
```

I am just wondering if `backend` is the easiest wording to understand for users so suggesting a slightly simpler alternative.

Reply:
I changed it to `The push is in progress and will continue running remotely. We will send you an email once it's completed.` in https://github.com/Automattic/studio/pull/2182/commits/4da6eea563f855d7bdbb7c0b93c59bd268455cc9